adapter/internal/oasparser/envoyconf/routes_with_clusters.go

```diff
@@ -833,3 +835,16 @@ func createAddress(remoteHost string, port uint32) *corev3.Address {
 	}}
 	return &address
 }
+
+// getMaxStreamDuration configures a maximum duration for a websocket route.
+func getMaxStreamDuration(apiType string) *routev3.RouteAction_MaxStreamDuration {
+	var maxStreamDuration *routev3.RouteAction_MaxStreamDuration = nil
+	if apiType == mgw.WS {
+		maxStreamDuration = &routev3.RouteAction_MaxStreamDuration{
+			MaxStreamDuration: &durationpb.Duration{
+				Seconds: 60 * 60 * 24,
```

Comment:
configured 24 hrs as `MaxStreamDuration`.

Reply:
Is this something we should read from the config

Reply:
We can do that too. Shall I read from the config ? This value will decide how long the web socket connection is kept open. Setting it to 24 hrs means, the connection will be automatically closed after 24 hrs.